Dockerfile

```
@@ -0,0 +1,17 @@
+FROM node:16-slim
```

Forge fails, it needs `GLIBC_2.29`
```suggestion
FROM node:18-slim
```